legal-api/src/legal_api/services/compliance/compliance_checks/__init__.py

```diff
@@ -0,0 +1,211 @@
+                    'message': 'Street is required for business office delivery address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_CITY: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_OFFICE_DELIVERY_ADDRESS_CITY,
+                    'message': 'City is required for business office delivery address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_COUNTRY: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_OFFICE_DELIVERY_ADDRESS_COUNTRY,
+                    'message': 'Country is required for business office delivery address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_POSTAL_CODE: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_OFFICE_DELIVERY_ADDRESS_POSTAL_CODE,
+                    'message': 'Postal code is required for business office delivery address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_REGION: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_OFFICE_DELIVERY_ADDRESS_REGION,
+                    'message': 'Region is required for business office delivery address.'
+                }
+            }
+        }
+    },
+    ComplianceWarningReferers.BUSINESS_PARTY: {
+        'addresses': {
+            'mailing': {
+                ComplianceWarnings.NO_ADDRESS: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS,
+                    'message': 'Business party mailing address is required.'
+                },
+                ComplianceWarnings.NO_ADDRESS_STREET: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS_STREET,
+                    'message': 'Street is required for business party mailing address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_CITY: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS_CITY,
+                    'message': 'City is required for business party mailing address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_COUNTRY: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS_COUNTRY,
+                    'message': 'Country is required for business party mailing address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_POSTAL_CODE: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS_POSTAL_CODE,
+                    'message': 'Postal code is required for business party mailing address.'
+                },
+                ComplianceWarnings.NO_ADDRESS_REGION: {
+                    'code': ComplianceWarningCodes.NO_BUSINESS_PARTY_MAILING_ADDRESS_REGION,
+                    'message': 'Region is required for business party mailing address.'
+                }
+            }
+        }
+    },
+    ComplianceWarningReferers.COMPLETING_PARTY: {
+        'addresses': {
```

Comment:
Why are we validating Completing Party? It is against filing and shouldnt have any impact on business compliance as fas as I know

Reply:
It is included here https://github.com/bcgov/entity/issues/12038

Reply:
We should check this, I believe Lekshmi is right in that so far completing parties are a filing level thing and not a business item.

Reply:
Confirmed with Linda in standup.  She still wants us to check for the completing party as staff need to be able to determine who the completing party was for the registration.